### PR TITLE
Fix versioning actions

### PR DESCRIPTION
**Pull request by scme0**
Fixes issues with the github actions... I wasn't meant to merge the other PR 🤦‍♂️

### CODE REVIEW

**Comment by scme0** [comment]
I'm keeping the actual publish step commented out until we can confirm that we get the right tags on master.

Once this is merged, I'll create a Versioning PR to test that part of it too and then I'll uncomment the publish step.

File: .github/workflows/build-and-push-container-image.yml [hunk 1]
```diff
@@ -29,15 +37,15 @@ jobs:
...
         run: |
-            if [ ${{ steps.isVersioningCommit.outputs.isVersioningCommit }} ]; then
+            if ${{ steps.isVersioningCommit.outputs.IS_VERSIONING_COMMIT }}; then
                 tagVersion=${{ steps.version.outputs.VERSION }}
-                tags="octopusdeploy/nfs-server:$tagVersion,octopusdeploy/nfs-server:latest";
+                tags="octopusdeploy/nfs-server:$tagVersion,docker.packages.octopushq.com/octopusdeploy/nfs-server:$tagVersion,octopusdeploy/nfs-server:latest,docker.packages.octopushq.com/octopusdeploy/nfs-server:latest";
             else
                 tagVersion=$(git describe --tags)
-                tags="octopusdeploy/nfs-server:$tagVersion,docker.packages.octopushq.com/octopusdeploy/nfs-server:$tagVersion,octopusdeploy/nfs-server:latest,docker.packages.octopushq.com/octopusdeploy/nfs-server:latest";
+                tags="docker.packages.octopushq.com/octopusdeploy/nfs-server:$tagVersion";
```
Comment (scme0) [inline .github/workflows/build-and-push-container-image.yml:46]: No we're only publishing the git describe versioned image to artifactory when it's not a release build.

When it's a release build we publish it as the release version and `latest` to docker hub and artifactory.

File: .github/workflows/build-and-push-container-image.yml [hunk 2]
```diff
@@ -29,15 +37,16 @@ jobs:
 
       - name: Get Tags
         run: |
-            if [ ${{ steps.isVersioningCommit.outputs.isVersioningCommit }} ]; then
+            if ${{ steps.isVersioningCommit.outputs.IS_VERSIONING_COMMIT }}; then
                 tagVersion=${{ steps.version.outputs.VERSION }}
-                tags="octopusdeploy/nfs-server:$tagVersion,octopusdeploy/nfs-server:latest";
+                tags="octopusdeploy/nfs-server:$tagVersion,docker.packages.octopushq.com/octopusdeploy/nfs-server:$tagVersion,octopusdeploy/nfs-server:latest,docker.packages.octopushq.com/octopusdeploy/nfs-server:latest";
             else
+                : # git describe --tags produces a value matching <tag>-<commits since tag>-<8 char commit hash>
```
Comment (scme0) [inline .github/workflows/build-and-push-container-image.yml:44]: `: #` for comments in `run: |` commands pinched from here: https://stackoverflow.com/a/68421196

**Review by flin-8 — APPROVED** [review]
LGTM

**Pull request merged**